serverless.yml

```diff
@@ -66,7 +66,22 @@ region: eu-north-1
 #    - include-me.js
 #    - include-me-dir/**
 
-#functions:
+functions:
+  getProductsList:
+    handler: src/server/products/getProductsList.getProductsListHandler
```

Comment:
Seems like the backend part is supposed to be located in different repo. Moreover, I don't see handler `src/server/products/getProductsList.getProductsListHandler` in this repo.